src/xdocs/config_coding.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<document xmlns="http://maven.apache.org/XDOC/2.0"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/XDOC/2.0 https://maven.apache.org/xsd/xdoc-2.0.xsd">
+
+  <head>
+    <title>Coding</title>
+  </head>
```

Comment:
Can we do title  as "Redirection to xxxxx" and in our ja code skip calculations based on page title 
https://stackoverflow.com/a/1057065/1015848

Reply:
I have changed the page titles to `config_coding` & `config_header`, these title helps us in redirection. For better explanation, pls refer https://github.com/checkstyle/checkstyle/pull/15949#discussion_r1867078104

Reply:
We need some word marker that will help us to return quickly from redirection function, it  it executed in active page.

Reply:
Can you elaborate it more? Im not able to understand it.

Reply:
Title should be `Redirection to coding.html`
`If (!title.startWith("Redirection to ") ) return;`

Reply:
done.